test/utils.js

```diff
@@ -9,22 +8,19 @@ const Level = require('level')
 const Path = require('path')
 const facilityCache = require('../lib')
 const testConfig = require('../lib/config')
+const assert = require('assert')
 
-const EXPECTED_FACILITY = ['654321', 'existing', 'za MomConnect Existing']
-const EXPECTED_HEADERS = [
-  {name: 'value', column: 'value', type: 'java.lang.String', hidden: false, meta: false},
-  {name: 'uid', column: 'uid', type: 'java.lang.String', hidden: false, meta: false},
-  {name: 'name', column: 'name', type: 'java.lang.String', hidden: false, meta: false}
-]
-
-const expect = Lab.assertions
-const lab = exports.lab = Lab.script()
+const EXPECTED_FACILITY = {
+  code: '654321',
+  displayName: 'za MomConnect Existing',
+  id: 'facility1'
+}
 
 const config1 = {
   routes: [{
     'dhisUrl': 'http://localhost:8000',
     'dhisPath': '/api/sqlViews/1',
```

Comment:
I think we can update this url and the few below to point to the OrganisationUnits endpoints as we are no longer make use of a sqlView.